### PR TITLE
Add Facebook SimpleQuestionV2

### PR DESCRIPTION
**Pull request by abhishekkrthakur**
Add simple questions v2: https://research.fb.com/downloads/babi/

### CODE REVIEW

**Comment by lhoestq** [comment]
I think @thomwolf may also be working on this one as part of the Babi benchmark in #945

**Review by thomwolf — APPROVED** [review]
Yes this one is good. Each dataset in the babi project can be an independant dataset so no conflict with my PR.

File: datasets/simple_questions_v2/simple_questions_v2.py [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+      title={Large-scale Simple Question Answering with Memory Networks},
+      author={Antoine Bordes and Nicolas Usunier and Sumit Chopra and Jason Weston},
+      year={2015},
+      eprint={1506.02075},
+      archivePrefix={arXiv},
+      primaryClass={cs.LG}
+}
+"""
+
+_URL = "https://uc7042fc5e1d2d9c831a318b49e7.dl.dropboxusercontent.com/cd/0/get/BEVJ4i_TOAt1pM8VdyJa0ctwmjEerlbDzNw2u8wXoMjY4s6WbgiKd3CA3gJ1deDcCR-5yyUKdpL0zS76uTuGcjX8To48_vwLjyeJwej8sbVBlg/file?_download_id=6436861371570024931106653846455288758893761540515299505641721092&_notify_domain=www.dropbox.com&dl=1"
```
Comment (lhoestq) [inline datasets/simple_questions_v2/simple_questions_v2.py:44]: Could you use this url instead ?
```
https://www.dropbox.com/s/tohrsllcfy7rch4/SimpleQuestions_v2.tgz?dl=1
```
I feel like the current one might have a limited lifetime

After updating the url you'll have to regenerate the dataset_infos.json

**Review by lhoestq — APPROVED** [review]
LGTM !
thanks for updating the url :)

**Pull request merged**